docs/conf.py

```diff
@@ -157,6 +157,7 @@ def get_xsversion(self):
     # notebooks
     'nbsphinx',
     'sphinx_rtd_theme',
+    'pytest_doctestplus.sphinx.doctestplus',
```

Comment:
No new requirement. `pytest-doctestplus` is part of the `sphinx-astropy` metapackage that we already depend on.